mygraph.py

```diff
@@ -78,12 +77,13 @@ def _add_incidence(self, edge: "Edge"):
         self._incidence[other].add(edge)
 
     @property
-    def graph(self) -> "Graph":
-        """
-        The graph of this vertex
+    def graphs(self) -> Set['Graph']:
+        """The graph this vertex has been added to.
+
```

Comment:
Waarom doe je dit?

Reply:
Wat bedoel je met dit? De docstring achter `"""Docstring here` plaatsen? [Dit doe ik volgens deze richtlijn.](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)

Reply:
Anders staat de eerste zin van de docstring in feite na een newline character. Ja. Python is zo'n taal met dit soort lelijkheden. 😱

Reply:
Nee ik bedoel eigenlijk een set van graphs maken, maar kwam er later al achter